website/content/en/v0.22.0/concepts/node-templates.md

```diff
@@ -334,6 +334,7 @@ spec:
   userData: |
     MIME-Version: 1.0
     Content-Type: multipart/mixed; boundary="BOUNDARY"
+
```

Comment:
Thanks for catching this!  Can you apply this change to `website/content/en/preview/concepts/node-templates.md` instead?  We promote the preview docs when we cut a release.

Reply:
hey @bwagner5 of course, but you confused me a bit ... apply the change to `preview` **instead** or **as well**?

Reply:
hey @tzneal took me a while as I was off for a few days, but I rebased and pushed now :)